liota/dcc/vrops.py

```diff
@@ -128,8 +128,10 @@ def on_receive_safe(msg):
                             try:
                                 uuid_path = config.get('UUID_PATH', 'uuid_path')
```

Comment:
Need to rename UUID_PATH and uuid.ini filename as it contains more details now, not just UUID.